config/webpack.common.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path')
```

Comment:
It may be nice, especially for future reference, to comment on the intention behind some of the settings and rules in this file and other `config` files like it

Reply:
I plan to go through and comment everything next time I have a chance to work on it.